update_command.php

```diff
@@ -219,5 +223,13 @@
     }
     output_repos_with_prs_created();
     output_prs_created();
+
+    // Report about any repos for which we couldn't find the right branch.
+    if (count($reposMissingBranch)) {
+        $reposString = implode("\n- ", $reposMissingBranch);
+        warning("Could not find branch to checkout for the following repos using --branch=$branchOption:\n- $reposString");
+        return Command::FAILURE;
+    }
```

Comment:
Looks like this at the end of execution (after all other output):

![image](https://github.com/silverstripe/module-standardiser/assets/36352093/132e73e7-f423-45b3-9166-d7dedadb0354)